mscore/palette.h

```diff
@@ -95,6 +95,7 @@ class PaletteScrollArea : public QScrollArea {
       PaletteScrollArea(Palette* w, QWidget* parent = 0);
       bool restrictHeight() const { return _restrictHeight; }
       void setRestrictHeight(bool val) { _restrictHeight = val; }
+      virtual void keyPressEvent(QKeyEvent* event) override;
```

Comment:
Keep it `protected` instead of `public` ?

Except this, it's good to merge. Check with @MarcSabatella but for me, next step would be the master palette because it's very close.